content/opt/setup/graphite

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Graphite writer
+if [ "${ICINGA2_FEATURE_GRAPHITE}" == "true" ] || [ "${ICINGA2_FEATURE_GRAPHITE}" == "1" ]; then
+	echo "=> Enabling Icinga2 graphite writer"
+
+	icinga2 feature enable graphite
+
+	cat >/etc/icinga2/features-available/graphite.conf <<-END
+	/**
+	 * The GraphiteWriter type writes check result metrics and
+	 * performance data to a graphite tcp socket.
+	 */
+	library "perfdata"
+	object GraphiteWriter "graphite" {
+	  host = "$ICINGA2_FEATURE_GRAPHITE_HOST"
+	  port = "$ICINGA2_FEATURE_GRAPHITE_PORT"
+	  enable_send_thresholds = true
+	}
+	END
+
+else
+	# Actively disable graphite, to not hit any weird bugs
+	icinga2 feature disable graphite
```

Comment:
Piping to /dev/null is a good idea.  I have a feeling the behaviour from the icinga2 CLI command is expected